flume-ng-channels/flume-file-channel/src/main/java/org/apache/flume/channel/file/instrumentation/FileChannelCounter.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.flume.channel.file.instrumentation;
+
+import org.apache.flume.channel.file.FileChannel;
+import org.apache.flume.instrumentation.ChannelCounter;
+
+/**
```

Comment:
Shouldn't license be the first thing of every source file? If yes than it is interesting that it passed RAT check.